.rubocop.yml

```diff
@@ -0,0 +1,18 @@
+AllCops:
```

Comment:
Hey @emaiax , thank you for cleaning this up.

Do you have any particular reason for picking this settings? 

I would prefer if we go with the [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide#the-ruby-style-guide) guidelines.

I think [RuboCop](https://github.com/bbatsov/rubocop) does many of them by default:

> RuboCop is a Ruby static code analyzer. Out of the box it will enforce many of the guidelines outlined in the community Ruby Style Guide.